Tchap/Generated/Strings.swift

```diff
@@ -3,456 +3,456 @@
 
 import Foundation
 
-// swiftlint:disable superfluous_disable_command file_length implicit_return
+// swiftlint:disable superfluous_disable_command file_length implicit_return prefer_self_in_static_references
```

Comment:
Could we ignore this file in git repo ?

Reply:
It is a good question.
For the moment, I think it is better to have the same behavior as Element, but we may discuss about this in a near future.